mage_integrations/mage_integrations/destinations/base.py

```diff
@@ -118,6 +119,10 @@ def config(self) -> Dict:
             self.logger.exception(message)
             raise Exception(message)
 
+    @config.setter
+    def config(self, config):
```

Comment:
where is it used?

Reply:
@wangxiaoyou1993 this is a bit a hack because the transformer class inherits from both `Source` and `Destination`. The source class sets the `config` variable, but it won't be able to set it in the transformer if the destination class's `config` is immutable. Since the transformer only is used as either a source or destination at the moment, I left it like this.

Reply:
ok

Reply:
could you add comment for this?